lib/Runtime/PlatformAgnostic/Intl.h

```diff
@@ -78,11 +106,16 @@ namespace Intl
     void SetNumberFormatGroupingUsed(_In_ IPlatformAgnosticResource *resource, _In_ const bool isGroupingUsed);
 
     template <typename T>
-    const char16 *FormatNumber(IPlatformAgnosticResource *formatter, const T val, const NumberFormatStyle formatterToUse, const NumberFormatCurrencyDisplay currencyDisplay, const char16 *currencyCode);
+    const char16 *FormatNumber(IPlatformAgnosticResource *formatter, const T val, const NumberFormatStyle formatterToUse,
+        const NumberFormatCurrencyDisplay currencyDisplay, const char16 *currencyCode);
 
-    bool ResolveLocaleLookup(_In_z_ const char16 *locale, _Out_ char16 *resolved);
```

Comment:
> bool ResolveLocaleLookup(_In_z_ const char16 *locale, _Out_ char16 *resolved); [](start = 3, length = 79)

Where did ResolveLocaleLookup go?

Reply:
the logic for resolveLocaleLookup is now implemented in Javascript, which only ends up calling platform.isLocaleAvailable instead.